tests/conftest.py

```diff
@@ -1166,6 +1166,31 @@ def ctx(create=False):
     return ctx
 
 
+@pytest.fixture()
+def s3_test_data_point_clouds(monkeypatch_session):
+    """
+    You can run tests that fetch a copy of the auckland test data from S3 (and so test Kart's S3 behaviour)
+    by setting KART_S3_TEST_DATA_POINT_CLOUDS=s3://some-bucket/path-to-auckland-tiles/*.laz
+    The tiles hosted there should be the ones found in tests/data/point-cloud/laz-auckland.tgz
+    """
+    if "KART_S3_TEST_DATA_POINT_CLOUDS" not in os.environ:
+        raise pytest.skip(
+            "S3 tests require configuration - read docstring at conftest.s3_test_data_point_clouds"
```

Comment:
we should stick the files in a public bucket and then we can just enable these tests unconditionally

Reply:
Done on CI but still using this variable - feels like its better than hard-coding it into the python.